modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro

```diff
@@ -52,6 +64,12 @@ definition {
 		AppBuilderAdmin.gotoBack();
 	}
 
+	macro addFromObjectField {
+		var key_fieldName = "${fieldName}";
+
+		DoubleClick(locator1 = "FormViewBuilder#OBJECT_FIELD_NAME");
+	}
```

Comment:
Hey @nkkwon , I would suggest you to create a new macro that will validate that the addFromObjectField macro was able to add the field to the form view. You will just check if the field was added to the form view body.